ui/src/test/java/edu/wpi/grip/ui/util/deployment/HeadlessInstanceManagerTest.java

```diff
@@ -0,0 +1,41 @@
+package edu.wpi.grip.ui.util.deployment;
+
+
+import com.google.common.eventbus.EventBus;
+import org.junit.Before;
+import org.junit.Ignore;
+import org.junit.Test;
+
+import java.io.File;
+import java.io.IOException;
+import java.net.URISyntaxException;
+
+import static junit.framework.TestCase.assertFalse;
+
+public class HeadlessInstanceManagerTest {
+
+    private HeadlessInstanceManager headlessInstanceManager;
+
+    @Before
+    public void setUp() throws URISyntaxException, IOException {
+
+        SecureShellDetails details = new SecureShellDetails.FRCSecureShellDetails(191);
+        DeploymentCommands deploymentCommands = new DeploymentCommands.FCRDeploymentCommands();
+        File coreJar = new File(edu.wpi.grip.core.Main.class.getProtectionDomain().getCodeSource().getLocation().toURI().getPath());
+        File testSave = new File(HeadlessInstanceManager.class.getResource("LoadThreeStepsTestSave.grip").toURI());
+
+        headlessInstanceManager = new HeadlessInstanceManager(new EventBus(), coreJar, testSave, details, deploymentCommands);
+    }
+
+    @Test(timeout = 30000)
+    @Ignore("This requires a robot to test on")
```

Comment:
This should not be an ignored unit test.  It's okay to write something like this for your own testing purposes, but this is useless to us as a unit test because we can't run it, and it makes it seem like our code is broken since we ignore it.